scripts/kernbench/dependencies.txt

```diff
@@ -0,0 +1,27 @@
+### START - Dependency installation order ###
+kernbench-order = 93
+### END - Dependency installation order ###
+
+### START - Dependency-specific installation method ###
+# pm = "package manager" (yum or apt-get)
+# sl = "soft link" (assume that the dependency is already installed, just has to
+# be properly exposed to the user's path.
+# git = git clone using above giturl
+# pip = python pip utility
+# man = "manual"
+kernbench-install = man
```

Comment:
Man needs to be changed to github so that the tool can automatically install the tool for the user.

Reply:
With the right URL